### PR TITLE
Clear reconnect timeout 

### PR DESCRIPTION
**Pull request by michax**
Clearing reconnect timeout upon connection and multiple consecutive reconnects.

Currently, reconnect timeouts are left hanging after they are fired. In some scenarios this can cause unnecessary firing of reconnects logic.

### CODE REVIEW

File: src/openapi/streaming/streaming.js [hunk 1]
```diff
@@ -159,6 +171,7 @@ function onConnectionStateChanged(change) {
             // if *we* are reconnecting (as opposed to signal-r reconnecting, which we do not need to handle specially)
             if (this.reconnecting) {
                 resetAllSubscriptions.call(this);
+                clearReconnect.call(this);
                 this.reconnecting = false;
```
Comment (lukeapage) [inline src/openapi/streaming/streaming.js]: Unit test?

**Comment by lukeapage** [comment]
@michax what happened with this Pr?

**Comment by lukeapage** [comment]
@michax what happened with this? It looks like the kind of change I'd like..

**Comment by michax** [comment]
Missed notification. Will take care of it.

**Pull request merged**